x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/action/RestGetLifecycleAction.java

```diff
@@ -36,11 +35,7 @@ public String getName() {
     @Override
     protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient client) {
         String[] lifecycleNames = Strings.splitStringByCommaToArray(restRequest.param("name"));
-        GetLifecycleAction.Request getLifecycleRequest = new GetLifecycleAction.Request(
-            getMasterNodeTimeout(restRequest),
-            getAckTimeout(restRequest),
-            lifecycleNames
-        );
+        GetLifecycleAction.Request getLifecycleRequest = new GetLifecycleAction.Request(getMasterNodeTimeout(restRequest), lifecycleNames);
```

Comment:
Shouldn't this be using `getAckTimeout(restRequest)` instead of the "master" variant, since it no longer goes through the master node?

Reply:
The master timeout is still relevant as we use that when we're waiting for any cluster blocks to be removed. The ack timeout was never used in this request as it was a simple GET request.